src/utils/export-map.ts

```diff
@@ -90,6 +90,38 @@ const parseComment = (comment: string): commentParser.Block => {
   }
 }
 
+function getTsconfigWithContext(context: ChildContext | RuleContext) {
+  const parserOptions = context.parserOptions || {}
+  let tsconfigRootDir = parserOptions.tsconfigRootDir
+  const project = parserOptions.project
+  const cacheKey = stableHash({ tsconfigRootDir, project })
```

Comment:
```suggestion
  const cacheKey = stableHash([tsconfigRootDir, project])
```

We didn't do this right last time, so we should do it right this time. Plain objects are more expensive than arrays.

Reply:
It's not related to this PR, a seperate PR would be fine.